doc/cephadm/osd.rst

```diff
@@ -392,24 +396,27 @@ Create a file called i.e. osd_spec.yml
     data_devices:                    <- the type of devices you are applying specs to
       all: true                      <- a filter, check below for a full list
 
-This would translate to:
+This means :
 
-Turn any available(ceph-volume decides what 'available' is) into an OSD on all hosts that match
-the glob pattern '*'. (The glob pattern matches against the registered hosts from `host ls`)
-There will be a more detailed section on host_pattern down below.
+1. Turn any available (ceph-volume decides what 'available' is) into an OSD on
```

Comment:
```suggestion
#. Turn any available (ceph-volume decides what 'available' is) into an OSD on
```

Reply:
Done.

I've also repaired the prompt that immediately follows this. (Previously, it duplicated the prompt in the command.)